.github/workflows/enos-ci-cleanup.yml

```diff
@@ -0,0 +1,92 @@
+name: enos-ci-cleanup
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '05 02 * * *'
+  push:
+    branches:
+      - "jb/QTI-400"
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      regions: ${{steps.setup.outputs.regions}}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CI }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CI }}
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+      - name: Get all regions
+        id: setup
+        run: |
+          echo "regions=$(aws ec2 describe-regions --region us-east-1 --output json --query 'Regions[].RegionName' | tr -d '\n ')" >> $GITHUB_OUTPUT
+
+  aws-nuke:
+    needs: setup
+    runs-on: ubuntu-latest
+    container:
+      image: rebuy/aws-nuke
+      options:
+         --user root
+         -t
+      env:
+        AWS_ACCESS_KEY_ID: ${{ env.AWS_ACCESS_KEY_ID }}
```

Comment:
This seems like it's self referencing. Same for the next line. Do we need this at all?

Reply:
unfortunately, yeah, this `env` is whats getting passed to Docker, it doesn't inherit any of the parent environment